### ISSUE TITLE
Training SSD  with the default "ssd312_coco.py" got the ap with 21.4% or 17.2%

### ISSUE THREAD

**Issue by convolutionROC** [open-mmlab/mmdetection#246]
I have trained the SSD three times with the default "ssd312_coco.py", but I got the ap with 21.4% or 17.2%, which was much worse than the result reported in the website. I noticed that the final training loss was still high, especailly the loss_cls:
2019-01-08 05:48:27,584 - INFO - Epoch [24][8750/9162]	lr: 0.00002, eta: 0:02:20, time: 0.332, data_time: 0.056, loss_cls: 3.1586, loss_reg: 1.4061, loss: 4.5647
2019-01-08 05:48:43,900 - INFO - Epoch [24][8800/9162]	lr: 0.00002, eta: 0:02:03, time: 0.326, data_time: 0.012, loss_cls: 3.2230, loss_reg: 1.3756, loss: 4.5986
2019-01-08 05:49:00,328 - INFO - Epoch [24][8850/9162]	lr: 0.00002, eta: 0:01:46, time: 0.329, data_time: 0.016, loss_cls: 3.1475, loss_reg: 1.3759, loss: 4.5234
2019-01-08 05:49:17,189 - INFO - Epoch [24][8900/9162]	lr: 0.00002, eta: 0:01:29, time: 0.337, data_time: 0.010, loss_cls: 3.2146, loss_reg: 1.4394, loss: 4.6540
2019-01-08 05:49:33,600 - INFO - Epoch [24][8950/9162]	lr: 0.00002, eta: 0:01:12, time: 0.328, data_time: 0.011, loss_cls: 3.2378, loss_reg: 1.4598, loss: 4.6976
2019-01-08 05:49:49,969 - INFO - Epoch [24][9000/9162]	lr: 0.00002, eta: 0:00:55, time: 0.327, data_time: 0.011, loss_cls: 3.2824, loss_reg: 1.4521, loss: 4.7345
2019-01-08 05:50:05,979 - INFO - Epoch [24][9050/9162]	lr: 0.00002, eta: 0:00:38, time: 0.320, data_time: 0.015, loss_cls: 3.3184, loss_reg: 1.4483, loss: 4.7667
2019-01-08 05:50:22,065 - INFO - Epoch [24][9100/9162]	lr: 0.00002, eta: 0:00:21, time: 0.322, data_time: 0.061, loss_cls: 3.2654, loss_reg: 1.3634, loss: 4.6288
2019-01-08 05:50:38,960 - INFO - Epoch [24][9150/9162]	lr: 0.00002, eta: 0:00:04, time: 0.338, data_time: 0.011, loss_cls: 3.2334, loss_reg: 1.5277, loss: 4.7611

The experiment is based on Ubuntu16.04-cuda9.0-cudnn7 with 8 Titan GPUs.

**Comment by convolutionROC** [comment]
The pre-trained model vgg16 is downloaded from the pytorch modelzoo, which seems to be different from the model in the ssd312_coco.py. It may be the problem.

**Comment by yhcao6** [comment]
Do you mean "ssd300_coco.py"? We don't have config "ssd312_coco.py".

The loss is abnormal, for a reference, the loss at the same epoch and iteration is around as this:
<img width="968" alt="screen shot 2019-01-09 at 10 07 38 am" src="https://user-images.githubusercontent.com/22662425/50871489-a1444800-13f6-11e9-9316-30e5853c0d5f.png">

**Comment by yhcao6** [comment]
The backbone of SSD is not exactly the normal VGG16, detail structure you can refer to LiuWei's another paper ["ParseNet"]("https://arxiv.org/pdf/1506.04579.pdf")

**Comment by convolutionROC** [comment]
Thanks for your reply. The loss based on the new pre-trained model is normal.
